src/server/scripts/EasternKingdoms/zone_silverpine_forest.cpp

```diff
@@ -1,139 +1,2148 @@
+                        _randomSpellIndex = 0;
+
+                    DoCastVictim(spellId);
+
+                    _events.ScheduleEvent(EVENT_SPELL_DAMAGE, 5s);
+                    break;
+                }
+            }
+        }
+
+        if (!UpdateVictim())
+            return;
+        else
+            DoMeleeAttackIfReady();
+    }
+
+private:
+    EventMap _events;
+    uint8 _randomSpellIndex;
+};
+
+enum QuestTheWarchiefCometh
+{
+    QUEST_THE_WARCHIEF_COMETH = 26965,
+
+    NPC_LADY_SYLVANAS_WINDRUNNER_44365 = 44365,
+    NPC_AGATHA_44608 = 44608,
+    NPC_GRAND_EXECUTOR_MORTUUS = 44615,
+    NPC_MALE_FALLEN_HUMAN = 44592,
+    NPC_FEMALE_FALLEN_HUMAN = 44593,
+    NPC_PORTAL_FROM_ORGRIMMAR = 44630,
+    NPC_GARROSH_HELLSCREAM = 44629,
+    NPC_HIGH_WARLORD_CROMUSH = 44640,
+    NPC_HELLSCREEMS_ELITE = 44636,
+    NPC_QUEST_MONSTER_CREDIT = 44629,
+
+    SPELL_RAISE_FORSAKEN = 83173,
+    SPELL_AIR_REVENANT_ENTRANCE = 55761,
+    SPELL_SIMPLE_TELEPORT = 12980,
+
+    EVENT_START_ANIM,
+    EVENT_AGATHA_RAISE_FORSAKEN = 201,
+    EVENT_TALK_SEQUENCE = 301,
+    EVENT_SUMMON_PORTAL = 401,
+    EVENT_SUMMON_GARROSH,
+
+    ACTION_START_WALKING = 0,
+    ACTION_START_ANIM = 101,
+
+    PLAYER_GUID = 99999,
+
+    MOVE_CROMUSH_TO_SYLVANAS = 5405701,
+    MOVE_CROMUSH_TO_HOME = 5405702,
+    MOVE_GARROSH_TO_HOME = 5405703
+};
+
+Position const ElitePos[16] =
+{
+    { 1387.90f, 1029.71f, 53.21853f, 2.827433f },
+    { 1389.79f, 1024.51f, 53.20833f, 2.827433f },
+    { 1388.05f, 1026.91f, 53.20833f, 2.827433f },
+    { 1388.16f, 1020.88f, 53.25523f, 2.827433f },
+    { 1385.79f, 1025.99f, 53.22593f, 2.827433f },
+    { 1386.69f, 1023.26f, 53.24393f, 2.827433f },
+    { 1384.33f, 1022.04f, 53.28123f, 2.827433f },
+    { 1391.10f, 1027.73f, 53.20483f, 2.827433f },
+
+    { 1359.10f, 1046.55f, 52.97053f, 5.253441f },
+    { 1360.89f, 1051.81f, 53.19793f, 5.253441f },
+    { 1360.75f, 1048.84f, 53.12893f, 5.253441f },
+    { 1364.43f, 1053.62f, 53.29343f, 5.253441f },
+    { 1363.08f, 1048.15f, 53.22223f, 5.253441f },
+    { 1364.08f, 1050.84f, 53.29163f, 5.253441f },
+    { 1366.69f, 1050.31f, 53.34203f, 5.253441f },
+    { 1357.85f, 1050.12f, 52.99823f, 5.253441f }
+};
+
+Position const PortalPos[3] =
+{
+    { 1358.62f, 1054.72f, 53.1200f, 0.f },
+    { 1393.27f, 1021.20f, 53.2225f, 0.f },
+    { 1404.71f, 1063.73f, 60.5617f, 0.f }
+};
+
+Position const GarroshPos = { 1402.45f, 1061.62f, 60.56173f, 3.926991f };
+
+Position const CromushPos = { 1404.71f, 1063.73f, 60.5617f, 2.827433f };
+
+// Grand Executor Mortuus - 44615
+struct npc_silverpine_grand_executor_mortuus : public ScriptedAI
+{
+    npc_silverpine_grand_executor_mortuus(Creature* creature) : ScriptedAI(creature) { }
+
+    void QuestAccept(Player* player, const Quest* quest) override
+    {
+        if (quest->GetQuestId() == QUEST_THE_WARCHIEF_COMETH)
+        {
+            me->GetAI()->SetGUID(player->GetGUID(), PLAYER_GUID);
```

Comment:
just all `SetGUID` no need for me->GetAI()

same for DoAction

Reply:
Fixed.